front/src/applications/operationalStudies/components/SimulationResults/ChartHelpers/drawArea.ts

```diff
@@ -0,0 +1,47 @@
+import * as d3 from 'd3';
+import { isEmpty } from 'lodash';
+import { Chart } from 'reducers/osrdsimulation/types';
+import { AreaBlock } from '../SpeedSpaceChart/prepareData';
+
+/**
+ * Draw area for the SpeedSpaceChart or the SpaceCurvesSlopesChart
+ */
+const drawArea = (
+  chart: Chart,
+  classes: string,
+  dataSimulation: AreaBlock[],
+  groupID: string,
+  interpolation: 'curveMonotoneX' | 'curveLinear',
```

Comment:
As drawArea is used only at 3 places and that this type is never used anywhere, I thought it would be better to keep it like this to avoid creating and exporting a new type, what do you think ?